backend/app/tests/services/inquiry/test_get_inquiry_service.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+
+import pytest
+from sqlmodel import Session
+
+from app.models import Inquiry
+from app.services.inquiries import get_inquiry_by_id
+
+
+@pytest.fixture(name="single_inquiry", scope="module")
+def fixture_single_inquiry(db: Session) -> Inquiry:
+    single_inquiry = Inquiry(text="How's your work-life balance?")
+    db.add(single_inquiry)
+    db.commit()
+    return single_inquiry
+
+
+def test_getInquiryService_whenCalledWithInvalidId_shouldRaiseException(
+    db: Session,
+) -> None:
+    with pytest.raises(ValueError, match=r"Id must be a valid UUID"):
+        get_inquiry_by_id(session=db, inquiry_id=12345)  # ruff: noqa
```

Comment:
Related to my comment in `services/inquiries.py` `get_inquiry_by_id` method. I think testing the type of the inquiry_id is not necessary so long as we are passing in a valid id (an id that maps to an inquiry in the db).

Reply:
My intention was to check the case when the `get_inquiry_by_id` service was called with wrong `inquiry_id` type from the API. I think I can enforce API to always call the service with valid param types - I'll add this test on the API layer.

Ruff linter was also catching invalid type when I was intentionally passing wrong typed param in the test. Trying to tell Ruff to ignore it for this test seemed to not work.

Removing this test from service layer.

Reply:
In the API layer, doesnt pydantic validate the type?

Reply:
You are right. Looks like the API is already configured to return an error response when passed with invalid param, effectively preventing it from calling the service:
```json
{
  "detail": [
    {
      "type": "uuid_parsing",
      "loc": [
        "path",
        "inquiry_id"
      ],
      "msg": "Input should be a valid UUID, invalid length: expected length 32 for simple format, found 1",
      "input": "1",
      "ctx": {
        "error": "invalid length: expected length 32 for simple format, found 1"
      }
    }
  ]
}
```

In the API layer test, I will just check response code and the error message.